pkgs/development/python-modules/ingredient-parser-nlp/default.nix

```diff
@@ -0,0 +1,58 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  nix-update-script,
+
+  setuptools,
+
+  nltk,
+  python-crfsuite,
+  pint,
+  floret,
+
+  pytestCheckHook,
+  nltk-data,
+}:
+buildPythonPackage rec {
+  pname = "ingredient-parser-nlp";
+  version = "2.0.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "strangetom";
+    repo = "ingredient-parser";
+    tag = version;
+    hash = "sha256-i14RKBcvU56pDNGxNVBvvpQ65FCbitMIfvN5eLLJCWU=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    nltk
+    python-crfsuite
+    pint
+    floret
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+  ];
+
+  pythonImportsCheck = [
+    "ingredient_parser"
+  ];
+
+  # Needed for tests
+  env.NLTK_DATA = nltk-data.averaged_perceptron_tagger_eng;
+
+  passthru.updateScript = nix-update-script { };
```

Comment:
I don't think you need to specify this for Python packages, they all get an update script based on PyPI if I remember correctly.
LGTM otherwise, build is still running here.

Reply:
Thanks, I removed it